student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(age, classroom, name, parent_permission: false)
+    super(name: name, age: age, parent_permission: parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Since the `Student` class is inheriting from the `Person` class, the constructor parameters `name` and `parent_permissions` should have default values `"Unknown"` and `true` respectively